woocommerce/sensei-course-progress
# Updates dependency checker for Sensei 2 and updated PHP requirement

Brings the work from the Content Drip PR over to Sensei Certificates: https://github.com/woocommerce/sensei-content-drip/pull/146

Replaces #57 
Fixes #56

### Testing Instructions
- If activated, deactivate all versions of Sensei. 
- Remove `sensei_version` and `woothemes_sensei_version` options.
- If not active, activate this plugin on built version of this branch. Verify the Sensei dependency message shows on Plugins page and Dashboard page.
- Activate Sensei 1.12.x. Verify message disappears.
- Deactivate Sensei 1.12.x. Activate Sensei 2.x. Verify message is still gone.
- Deactivate Sensei 2.x and activate Sensei with WooCommerce Paid Courses. Verify message is still gone.

Test PHP version:
- Activate Sensei and version of this branch on PHP 5.2-5.5. Verify PHP version notice appears.
